ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -189,3 +208,83 @@ extension PillButtonBarDemoController: PillButtonBarDelegate {
         present(alert, animated: true)
     }
 }
+
+extension PillButtonBarDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        fluentTheme.register(tokenSetType: PillButtonTokenSet.self,
+                             tokenSet: isOverrideEnabled ? themeWideOverridePillButtonTokens : nil)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        self.bars.forEach({ bar in
+            let tokens = isOverrideEnabled ? perControlOverridePillButtonTokens : nil
+            bar.pillButtonOverrideTokens = tokens
+        })
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokens(for: PillButtonTokenSet.self) != nil
+    }
+
+    // MARK: - Custom tokens
+
+    private var themeWideOverridePillButtonTokens: [PillButtonTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .font: .uiFont {
+                return UIFont.fluent(FontInfo(name: "Times", size: 15.0, weight: .regular))
```

Comment:
Not sure if we want to keep this format or move forward with the new style @mischreiber 

For example from the ButtonDemoController:

.titleFont: .uiFont {
                return UIFont(descriptor: .init(name: "Papyrus", size: 20.0),
                              size: 20.0)
            },

Reply:
Yep, I can change it